src/components/media/VideoStream.module.css

```
@@ -24,3 +24,24 @@
     z-index: 1;
   }
 }
+
+.playbackPrompt {
+  position: fixed;
+  z-index: 1000;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  padding: 12px 18px;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)